adapter/internal/api/apis_impl.go

```diff
@@ -109,6 +119,23 @@ func ProcessMountedAPIProjects() (err error) {
 		}
 	}
 
+	payload, err := ioutil.ReadFile(apisDirName)
+	zipReader, err := zip.NewReader(bytes.NewReader(payload), int64(len(payload)))
+	if err != nil {
+		loggers.LoggerSync.Errorf("Error occured while unzipping the apictl project. Error: %v", err.Error())
+		return err
+	}
+
+	deploymentDescriptor, _, err := sync.ReadRootFiles(zipReader)
+	if err != nil {
+		loggers.LoggerAPI.Error("Error occured while reading root files ", err)
+		return err
+	}
+
+	if len(deploymentDescriptor.Data.Deployments) > 0 {
+		isPaidOrg = deploymentDescriptor.Data.Deployments[0].IsPaidOrg
+	}
+
```

Comment:
This func `ProcessMountedAPIProjects` is not getting hit in the CC with the Choreo path. This function was used to run CC in Standalone way. We can remove code added to this func.
```suggestion
```